build.sbt

```diff
@@ -102,6 +102,7 @@ lazy val includeGeneratedSrc: Setting[_] = {
 lazy val catsSettings = commonSettings ++ publishSettings ++ scoverageSettings ++ javadocSettings
 
 lazy val scalaCheckVersion = "1.13.5"
+lazy val scalaMockVersion = "3.6.0"
```

Comment:
Is it just me that finds mocks (in a functional context) a bit of a flag for things that could be improved?

Reply:
I fully agree, I think `Future` is good as a gateway drug, but I'm not sure we should encourage mocks. :)

Reply:
Okey, fixing now